Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -11,6 +11,7 @@ class RouterSpec: QuickSpec {
             guard let urlOne = URL(string: Resource.test_get.rawValue),
```

Comment:
These tests are still testing via the old style of tests. Don't forget to update these to match how we refactored tests on Friday.

Reply:
will take a look

Reply:
fixed